modules/fedimint-mint/src/lib.rs

```diff
@@ -154,10 +154,11 @@ impl ModuleGen for MintGen {
     async fn init(
         &self,
         cfg: ServerModuleConfig,
-        _db: Database,
+        db: Database,
         _env: &BTreeMap<OsString, OsString>,
         _task_group: &mut TaskGroup,
     ) -> anyhow::Result<DynServerModule> {
+        self.migrate_database(db).await?;
```

Comment:
I'm still not super happy with manually calling and defining `migrate_database`. 

My ideal: 
* let modules define a fn that returns the migration map
* Default-implement `migrate_database` on the trait
* Let `FedimintConsensus` call it (or init? idk tbh)

Reply:
Ha, I actually had it in the trait initially and moved it out. I can move it back.

> Let `FedimintConsensus` call it (or init? idk tbh)

I just realized that it needs to be called in `init`, since the database is passed in as a parameter there. Can't be called after since it is possible that the database is used in `init`, and can't be called before (if we want it on the trait) since the module isn't initialized yet.

Reply:
Moved into `ModuleGen` trait